[Bug]: Motion Notifications Not Working in Home App

### What happened?

RC3 - despite no errors and everything being set up correctly, so far as I can see, the home app on iOS or Mac will not register motion and trigger a notification. The Eufy app does and works. Ring notifications and streaming are working fine.

EDIT: Normal logging level shows this error: [8/24/2021, 7:11:35 AM] [homebridge-eufy-security] This plugin generated a warning from the characteristic 'Motion Detected': Characteristic not in required or optional characteristic section for service CameraOperatingMode. Adding anyway.. See https://git.io/JtMGR for more info.

### Plugin Version

v1.0.x (Supported)

### HomeBridge Version

v1.3.x (Supported)

### NodeJS Version

v14 (Supported)

### Operating System

Mac OS X

### Relevant log output

```shell
[2021-08-24T05:52:20.184Z] DEBUG: ef/4098 on v1.0.0-rc.3: Doorbell Property Changes:  --  personDetected { value: true, timestamp: 1629784339494 }
[2021-08-24T05:52:20.185Z] DEBUG: ef/4098 on v1.0.0-rc.3: Doorbell Property Changes:  --  personName { value: 'Unknown', timestamp: 1629784339494 }
[2021-08-24T05:52:20.185Z] DEBUG: ef/4098 on v1.0.0-rc.3: 
    Doorbell Property Changes:  --  pictureUrl {
      value: 'https://security-app-eu.eufylife.com/v1/s/g/[deletedbyme]',
      timestamp: 1629784339494
    }
[2021-08-24T05:52:20.187Z] DEBUG: ef/4098 on v1.0.0-rc.3: Doorbell Handle Camera motion: true
[2021-08-24T05:52:27.277Z] DEBUG: ef/4098 on v1.0.0-rc.3: Snapshot requested: 1280 x 720 Doorbell true
[2021-08-24T05:52:27.278Z] DEBUG: ef/4098 on v1.0.0-rc.3: Sending snapshot: 1280 x 720 Doorbell true
[2021-08-24T05:52:27.279Z] DEBUG: ef/4098 on v1.0.0-rc.3: Thumbnail URL:  -i https://security-app-eu.eufylife.com/v1/s/g/[deletedbyme]
[2021-08-24T05:52:27.305Z] DEBUG: ef/4098 on v1.0.0-rc.3: Snapshot command: /usr/local/lib/node_modules/homebridge-eufy-security/node_modules/ffmpeg-for-homebridge/ffmpeg -i https://security-app-eu.eufylife.com/v1/s/g/[deletedbyme] -frames:v 1 -filter:v scale=1280:720,scale='trunc(iw/2)*2':'trunc(ih/2)*2' -f image2 - Doorbell true
```

--- a/src/accessories/DoorbellCameraAccessory.ts
+++ b/src/accessories/DoorbellCameraAccessory.ts
@@ -16,9 +16,7 @@ import { CameraAccessory } from './CameraAccessory';
  */
 export class DoorbellCameraAccessory extends CameraAccessory {
 
-  protected service: Service;
   protected DoorbellCamera: DoorbellCamera;
-  protected motion_triggered: boolean;
 
   private doorbellService: Service;
 
@@ -31,17 +29,6 @@ export class DoorbellCameraAccessory extends CameraAccessory {
     this.DoorbellCamera = eufyDevice;
 
     this.platform.log.debug(this.accessory.displayName, 'Constructed Doorbell');
-
-    this.motion_triggered = false;
-
-    this.service =
-      this.accessory.getService(this.platform.Service.CameraOperatingMode) ||
-      this.accessory.addService(this.platform.Service.CameraOperatingMode);
-
-    this.service.setCharacteristic(
-      this.platform.Characteristic.Name,
-      accessory.displayName,
-    );
 
     this.doorbellService =
       this.accessory.getService(this.platform.Service.Doorbell) ||

